### ISSUE TITLE
Text is bolder in screenshots than in the original sim

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/graphing-quadratics#163]
**Test device**
MacBook Air (m1 chip) and iMac

**Operating System**
11.4 and 10.15.7

**Browser**
Safari and chrome

**Problem description**
https://github.com/phetsims/qa/issues/692

This is more of an FYI-- when taking screenshots in the Screenshot Wrapper, the text in the screenshot appears bolder than in the original. Everything else is the same aside from that. I also noticed this happening when selecting the option in the PhET menu in the sim. 
This did not happen using a chromebook. Nor did it occur with mac + firefox. 

**Visuals**
All of the equations in the graph area are bolder. The differences are most obvious in the Quadratic Terms accordion box. 

Sim:
<img width="871" alt="Screen Shot 2021-08-20 at 10 06 53 AM" src="https://user-images.githubusercontent.com/87318828/130257434-a0bb1389-e696-4656-add8-92e544391bae.png">

Screenshot:
<img width="871" alt="Screen Shot 2021-08-20 at 10 07 10 AM" src="https://user-images.githubusercontent.com/87318828/130257479-98a1ebb9-b659-4833-9235-639fb1f381d0.png">

**Comment by pixelzoom** [comment]
@amanda-phet I don't think this is something that should block publication. Do you agree?

@Nancy-Salpepi a couple of questions...

(1) Is all text bolder, or only the text that's rendering math symbols?

(2) Can you reproduce this in other sims?  Or just in Graphing Quadratics?

**Comment by Nancy-Salpepi** [comment]
(1) All text is bolder--if you look at "quadratic terms" and "equations" you can see the difference here (screenshot on left)
<img width="697" alt="Screen Shot 2021-08-23 at 8 30 34 AM" src="https://user-images.githubusercontent.com/87318828/130447777-c21bd366-32cd-4a43-b3dc-9af1f795037f.png">

(2) I can reproduce with other sims. I checked pH scale screen shot wrapper and noticed the same thing:
<img width="661" alt="Screen Shot 2021-08-23 at 8 27 10 AM" src="https://user-images.githubusercontent.com/87318828/130448056-a4be68ae-3904-4a93-9757-43c8b4b16e6c.png">

It hasn't prevented me from being able to read anything or cause anything to come out of alignment/overlap from what I have seen.

**Comment by amanda-phet** [comment]
I don't think this needs to block publication for this sim. If it ever causes and issue with legibility I'd want to investigate, but I don't think we need to address this at this time.

**Comment by pixelzoom** [comment]
Since this is a general problem that affects all sims, I'm going to transfer this issue from graphing-quadratics to joist.  And I'll add it to developer meeting.
